### ISSUE TITLE
Unclear label for unblocking elements

### ISSUE THREAD

**Issue by foxbunny** [brave/brave-browser#5242]
## Description 

The Brave context menu says "Block element via selector". To me, a logical opposite of "block" is "unblock". Since I could not find such an item in the context menu, I looked in the settings, and on internet to no avail. 

I believe it would improve discovery of the unblock functionality if it were called "Unblock blocked elements ....", rather than "Clear CSS rules ...". Also, the documentation for this feature would be a lot more helpful if it would mention how to reverse the effects, as it's rather easy to make a mistake on sites that use weird auto-generated selectors.

## Steps to Reproduce

   1. n/a

## Brave version (brave://version info)

Brave | 0.66.99 Chromium: 75.0.3770.100 (Official Build) (64-bit)
-- | --
Revision | cd0b15c8b6a4e70c44e27f35c37a4029bad3e3b0-refs/branch-heads/3770@{#1033}
OS | Windows 10 OS Version 1903 (Build 18362.175)
JavaScript | V8 7.5.288.23

## Version/Channel Information:

- Can you reproduce this issue with the current release?
- Can you reproduce this issue with the beta channel?
- Can you reproduce this issue with the dev channel?
- Can you reproduce this issue with the nightly channel?

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by ChrisBrownie55** [comment]
Would "unblock elements" be a little bit better than "unblock blocked elements"?

**Comment by balaji303** [comment]
Even I have this issue. Please say how to unblock elements.

**Comment by bsclifton** [comment]
Closing as `wontfix` - we have a new cosmetic filter in place. I created https://github.com/brave/brave-browser/issues/8914 to track removing this older version
